packages/vue-app/template/server.js

```diff
@@ -219,7 +243,7 @@ export default async (ssrContext) => {
     return Promise.all(promises)
   }))
 
-  <% if (isDev) { %>if (asyncDatas.length) debug('Data fetching ' + ssrContext.url + ': ' + (Date.now() - s) + 'ms')<% } %>
+  <% if (debug) { %>if (asyncDatas.length) debug('Data fetching ' + ssrContext.url + ': ' + (Date.now() - s) + 'ms')<% } %>
```

Comment:
Do we need `debug` package anymore? We can use `consola.debug` here.

Reply:
Agree

Reply:
Will it be a breaking change?

Reply:
I don't think :) Usage is the same as before.